sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/ResourceConfiguration.java

```diff
@@ -32,12 +34,16 @@
  */
 public final class ResourceConfiguration {
 
+  private static final Logger logger = Logger.getLogger(ResourceConfiguration.class.getName());
+
   private static final AttributeKey<String> SERVICE_NAME = AttributeKey.stringKey("service.name");
 
   // Visible for testing
   static final String ATTRIBUTE_PROPERTY = "otel.resource.attributes";
   static final String SERVICE_NAME_PROPERTY = "otel.service.name";
-  static final String DISABLED_ATTRIBUTE_KEYS = "otel.experimental.resource.disabled.keys";
+  static final String EXPERIMENTAL_DISABLED_ATTRIBUTE_KEYS =
+      "otel.experimental.resource.disabled.keys";
+  static final String DISABLED_ATTRIBUTE_KEYS = "otel.resource.disabled.keys";
```

Comment:
Can you update this [unit test](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/ResourceConfigurationTest.java#L25-L47) to verify both the experimental and new property names work? Thanks!